R/config-log.R

```diff
@@ -86,7 +88,9 @@ config_log_impl <- function(.mods) {
 
   if (all(missing)) {
     warning(glue("Found no bbi_config.json files for {length(.mods)} models."), call. = FALSE)
-    return(tibble())
+    return(
+      c(ABS_MOD_PATH, RUN_ID_COL) %>% map_dfc(~ tibble(!!.x := character()))
```

Comment:
I agree that the symptom is the same (the `run` column is missing from the object returned to `add_log_impl()`), but I'm not sure the solution should be the same. In particular, if all models are missing configuration files, there is not a separate error message to present to the user (as might be the case if all model summaries fail), so I'm not sure returning an (effectively) empty tibble gives the user information they wouldn't get by throwing an error instead.